.github/workflows/sonarcloud.yml

```
@@ -20,26 +20,28 @@ jobs:
         with:
           python-version: 3.12.4
 
-      # Set up Node.js and Vitest
+      # Set up Node.js and Vitest (Frontend)
       - name: Set up Node.js
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)